### PR TITLE
Fixed map not found on cross platform play

### PR DESCRIPTION
**Pull request by Mailaender**
See https://github.com/OpenRA/OpenRAModSDK/issues/162.

### CODE REVIEW

File: mods/sa/maps/batteground/map.yaml [hunk 1]
```diff
@@ -223,12 +223,6 @@ Actors:
 		Owner: Multi1
 		Location: 10,7
 		TurretFacing: 92
-	Actor65: object_48
-		Owner: Neutral
-		Location: 30,33
-	Actor66: object_48
-		Owner: Neutral
-		Location: 31,29
```
Comment (Dzierzan) [inline mods/sa/maps/batteground/map.yaml:231]: Why did you remove Actor65 and 66?

Comment (Mailaender) [reply mods/sa/maps/batteground/map.yaml:231]: They don't exist in the rules.
